intercity-server.rb

```diff
@@ -20,11 +24,41 @@ def usage
     puts "Commands:"
     puts "    help - Show the commands available"
     puts "    install - Run the setup for installing intercity-server"
+    puts "    restart - Restart your Intercity instance"
+    puts "    update - Update your Intercity instance"
   end
 
   def install
+    if installed?
+      puts "Intercity is already installed."
```

Comment:
Perhaps add one line with. "If you want to remove/update intercity, run XXX" because that's usually a thing people expect if you run the bash script again.

But otherwise :+1: :D